cors/resources/status.py

```diff
@@ -1,37 +1,37 @@
 def main(request, response):
-    response.headers.set("Access-Control-Allow-Origin", request.headers.get("origin"))
-    response.headers.set("Access-Control-Expose-Headers", "X-Request-Method")
+    response.headers.set(b"Access-Control-Allow-Origin", request.headers.get(b"origin"))
+    response.headers.set(b"Access-Control-Expose-Headers", b"X-Request-Method")
 
-    if request.method == 'OPTIONS':
-        response.headers.set("Access-Control-Allow-Methods",  "GET, CHICKEN, HEAD, POST, PUT")
+    if request.method == u'OPTIONS':
+        response.headers.set(b"Access-Control-Allow-Methods",  b"GET, CHICKEN, HEAD, POST, PUT")
 
-    if 'headers' in request.GET:
-        response.headers.set("Access-Control-Allow-Headers",  request.GET.first('headers'))
+    if b'headers' in request.GET:
+        response.headers.set(b"Access-Control-Allow-Headers",  request.GET.first(b'headers'))
 
-    response.headers.set("X-Request-Method", request.method)
+    response.headers.set(b"X-Request-Method", request.method.encode("iso-8859-1"))
```

Comment:
Use the `isomorphic_encode` utility function here.

Reply:
Updated.